services/processor/native/repository/BenchmarkToken/contract.go

```diff
@@ -24,7 +24,7 @@ type contract struct{ *sdk.BaseContract }
 
 ///////////////////////////////////////////////////////////////////////////
 
-const TOTAL_SUPPLY = 10000
+const TOTAL_SUPPLY = 1000000
```

Comment:
This doesn't really solve the problem for the forever live test net
Eventually it will run out of tokens
It runs twice successfully but won't survive infinite runs

Reply:
Opened an issue https://github.com/orbs-network/orbs-network-go/issues/500